integration-libs/cdc/schematics/add-cdc/index.ts

```diff
@@ -37,13 +37,21 @@ export function addCdcFeature(options: SpartacusCdcOptions): Rule {
       addPackageJsonDependenciesForLibrary(peerDependencies, options),
 
       shouldAddFeature(CLI_CDC_FEATURE, options.features)
-        ? addCdc(options)
+        ? addCdc(options, _context)
         : noop(),
     ]);
   };
 }
 
-function addCdc(options: SpartacusCdcOptions): Rule {
+function addCdc(options: SpartacusCdcOptions, context: SchematicContext): Rule {
+  if (!options.jsSDKUrl) {
+    context.logger.warn(
+      `CDC JS SDK URL is not provided. Please run the schematic again, or make sure you update the jsSDKUrl.`
+    );
+  }
+
+  let jsSDKUrl = options.jsSDKUrl? options.jsSDKUrl: "<url-to-cdc-script>";
```

Comment:
Since we're not re-declaring the variable, `const` should be used.

However, as mentioned here `https://github.com/SAP/spartacus/pull/15756/files#r875015142`, we most likely do not have to check as it will be a required param (if ever, but with a default value) 
Basically, if they decide to just press enter and skip the interactive params setting, they'll get the default value.